int32 numberOfFacets
Impinge[] ImpingementArray
